tools/travis/osx_install.sh

```diff
@@ -1,8 +1,17 @@
 #!/bin/bash
+set -ex
+
 brew update
 brew install ccache
 brew tap caskroom/cask
 brew cask install basictex
+# updating libpng requires python to be updated first
+brew install python@2
```

Comment:
Just curious, what does `@2` mean?

Reply:
No idea. I don’t have a MAC but I was trying to force python 2 to get updated.